packages/apps-v5/test/commands/apps/rename.js

```diff
@@ -24,4 +24,20 @@ describe('heroku apps:rename', function () {
       .then(() => expect(cli.stdout).to.equal('https://foobar.com | https://git.heroku.com/foobar.git\n'))
       .then(() => api.done())
   })
+
+  it('gives a message if the web_url is still http', function() {
+    let api = nock('https://api.heroku.com')
+    .patch('/apps/myapp', { name: 'newname' })
+    .reply(200, {
+      name: 'foobar',
+      web_url: 'http://foobar.com'
+    })
+
+    return cmd.run({ app: 'myapp', flags: {}, args: { newname: 'newname' }, httpGitHost: 'git.heroku.com' })
+      .then(() => expect(unwrap(cli.stderr)).to.equal(`Renaming myapp to newname... done Don't forget to update git remotes for all other local checkouts of the app.
```

Comment:
This PR is great! The only thing is there seems to be additional spacing in this assertion when checking `cli.stderr`. which is why CI is failing.